core/src/main/scala/org/apache/spark/storage/StorageStatusListener.scala

```diff
@@ -17,26 +17,55 @@
 
 package org.apache.spark.storage
 
+import java.util.concurrent.TimeUnit
+
+import scala.collection.JavaConversions.collectionAsScalaIterable
```

Comment:
avoid using JavaConversions, you should prefer JavaConverters, which forces you to call `.asScala`, making the transformation much clearer to future code readers.  the convention is to `import scala.collection.JavaConverters._`